package.json

```diff
@@ -30,6 +31,7 @@
     "jquery-ui": "1.10.5",
     "jssha": "1.5.0",
     "jws": "*",
+    "left-pad": "1.1.3",
```

Comment:
@saghul genuine question: i know there was the big debacle with left-pad a while back, but what is the concern with it now? is its ownership/existence still in question?